CHANGES.md

```diff
@@ -28,6 +28,7 @@ Release Notes.
 * Implement Kafka Log Reporter. Add config item of `agnt.conf`, `plugin.kafka.topic_logging`.
 * Upgrade byte-buddy to 1.11.18
 * Add plugin to support Apache HttpClient 5.
+* Format Springmvc & Tomcat EntrySpan operation name to `METHOD:URI`
```

Comment:
```suggestion
* Format SpringMVC & Tomcat EntrySpan operation name to `METHOD:URI`
* Make `method name` in new operation name according to runtime, rather than previous code-level definition, which used to have possibilities including multiple HTTP methods.
```

Reply:
@wankai123 You missed my second one.